bin/dart.bat

```diff
@@ -0,0 +1,46 @@
+@ECHO off
+REM Copyright 2014 The Flutter Authors. All rights reserved.
+REM Use of this source code is governed by a BSD-style license that can be
+REM found in the LICENSE file.
+
+REM ---------------------------------- NOTE ----------------------------------
+REM
+REM Please keep the logic in this file consistent with the logic in the
+REM `flutter` script in the same directory to ensure that Flutter continues to
+REM work across all platforms!
+REM
+REM --------------------------------------------------------------------------
+
+SETLOCAL ENABLEDELAYEDEXPANSION
+
+FOR %%i IN ("%~dp0..") DO SET FLUTTER_ROOT=%%~fi
+
+REM Include shared scripts in shared.bat
+SET shared_bin=%FLUTTER_ROOT%/bin/internal/shared.bat
+CALL "%shared_bin%"
+
+SET flutter_tools_dir=%FLUTTER_ROOT%\packages\flutter_tools
```

Comment:
nit: some of the variables set here seem unused? e.g. snapshot_path and engine_stamp

Reply:
Removed